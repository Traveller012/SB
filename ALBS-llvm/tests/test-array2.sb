 { : int} main = [

  flt[] a;
/*
  flt b;
  b = -2.3;
*/
  a = new flt[5];

  a[0]= 1.9;
  a[2] = -2.3; 
/*
  print(1.9);

  print(-2.3);
  print(b);
*/
  print(a[0]);
  print(a[2]);
/*
  b = new int[len*2];
  i = 0;
  while (i < len*2) {
    a[i] = i * -2;
    i = i + 1;
  }
  i = 0;
  while (i < len*2) {
    print(a[i]);
    i = i + 1;
  }
  */

  rtn 0;
]
