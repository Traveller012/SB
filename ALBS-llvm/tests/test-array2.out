1.900000
-2.300000
